### DISCUSSION TITLE
Lampadina zigbee - ripetitore

### DISCUSSION THREAD

**Discussion by aleisdone** [indomus/forum#184] [HUB: Home Assistant]
Buongiorno a tutti,
Premetto che sono alle prime armi con la domotica e Home Assistant.

Ho recentemente installato deConz e ho un problema con una presa che rimane nascosta.
Vorrei quindi utilizzare una lampadina come ripetitore per far raggiungere dal segnale anche la presa.
Avete qualche consiglio sulla scelta della lampadina?
Ho visto on line le Osram e le Ledvance ma non ho capito se funzionano appunto come ripetitore.
Grazie!

**Comment by snakuzzo** [comment]
In linea di massima tutti i device Zigbee alimentati NON a batteria fungono da router.
Puoi consultare [QUESTA LISTA](https://zigbee.blakadder.com/deconz.html)

Considera comunque che la lampadina magari può non essere alimentata sempre. Se spegni l'interruttore fisico, la lampadina non è in tensione e quindi sarà spenta...non funzionerà

**Comment by KC-inDomus** [comment]
corretto quel che dice Diego: la lampadina, comunque, dovrai sempre accenderla/spegnerla non levandogli corrente a monte, ma logicamente.

**Comment by aleisdone** [comment]
Benissimo, grazie mille!
